### PR TITLE
Feat/manager 9096: improve storage container display info 

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-9093`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-9096
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
1. improve container display info
2. add guides section

### CODE REVIEW

File: packages/manager/modules/ng-layout-helpers/src/onboarding/onboarding.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <div class="container-fluid mx-0 px-0 manager-on-boarding-layout">
-    <div class="row align-items-center">
+    <div data-ng-if="!$ctrl.hideIllustrationSection"
+         class="row align-items-center">
         <div class="w-100">
             <div class="placeholder p-5 d-flex justify-content-center">
```
Comment (zakariasahmane) [inline packages/manager/modules/ng-layout-helpers/src/onboarding/onboarding.html:5]: Is this a new demand ? We just refactored all onboarding pages for the PNR project that is still in beta, i don't think this complies with the new onboarding pages format.

**Comment by zakariasahmane** [comment]
Also please change the commit message from `improve container display info` to `improve storage containers display info` as the word `container` now has another meaning and is another app that is important in the `uFrontEnd` architecture.

**Review by zakariasahmane — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/containers/containers.constants.js [hunk 2]
```diff
@@ -96,6 +96,21 @@ export const OPENIO_DEFAULT_REGION = 'SBG';
...
+  {
+    id: 'veeam',
+    link: 'https://docs.ovh.com/fr/storage/s3/veeam/',
+  },
+  {
+    id: 's3-limitation',
+    link: 'https://docs.ovh.com/fr/storage/s3/limitations/',
+  },
+];
+
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/containers/containers.constants.js:113]: Is there only need for a FR guide?

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/containers/containers.constants.js:113]: No, you have right 👍 .
We are waiting for others links.

File: packages/manager/modules/ng-layout-helpers/src/onboarding/onboarding.html (hunk 1 above)
Comment (zakariasahmane) [reply packages/manager/modules/ng-layout-helpers/src/onboarding/onboarding.html:5]: Yes, i know, but all the empty pages are now supposed to have the same pattern, if no image, then there is a default image displayed, it's not hidden. 

I think you should talk with Mounir before doing this, as everything have been harmonized

**Review by vks-ovh — APPROVED** [review]
Please take care of the other subsidiary guides otherwise looks good to me.

File: packages/manager/modules/ng-layout-helpers/src/onboarding/onboarding.html (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/ng-layout-helpers/src/onboarding/onboarding.html:5]: Thanks for info.
I will discuss with him about this point 👍.

File: packages/manager/modules/pci/src/projects/project/storages/containers/containers.constants.js (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/project/storages/containers/containers.constants.js]: Done

**Review by ghyenne — APPROVED** [review]

**Pull request merged**